### PR TITLE
Css issues vuetify vue3 upgrade

### PR DESCRIPTION
**Pull request by MaelCormier-CGI**
# Description

Fixed CSS issues on Homepage, Header and Footer after migration to Vuetify 3.

## Types of changes

CSS updates

## Checklist

- [ ] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [ ] I have checked that unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: frontend/src/components/Header.vue [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <template>
-  <v-system-bar absolute color="rgb(0, 51, 102)" height="66rem" class="sysBar">
+  <v-system-bar absolute color="rgb(0, 51, 102)" class="sysBar" :style="{ height: '66px', textAlign: 'left' }" >
```
Comment (weskubo-cgi) [inline frontend/src/components/Header.vue:2]: Can we just move these styles into sysBar class?

File: frontend/src/App.vue [hunk 2]
```diff
@@ -4,15 +4,17 @@
     <HeaderComponent />
     <SnackBar />
     <NavBar v-if="pageTitle && isAuthenticated && showNavBar" :title="pageTitle" />
-    <v-main fluid class="align-start">
+    <v-main fluid style="--v-layout-top: 64px;" class="align-start">
+      <div class="v-main__wrap">
       <v-app-bar
         v-if="bannerColor !== ''"
-        style="color: white"
+        :style="{ height: '20px', color: 'white', position: 'relative', top: '0'}"
```
Comment (weskubo-cgi) [inline frontend/src/App.vue:11]: Given the additional styles, should we add a class for this?

**Review by weskubo-cgi — CHANGES_REQUESTED** [review]

File: frontend/src/components/Header.vue (hunk 1 above)
Comment (MaelCormier-CGI) [reply frontend/src/components/Header.vue]: Moved inline style to sysBar class

File: frontend/src/App.vue (hunk 2 above)
Comment (MaelCormier-CGI) [reply frontend/src/App.vue]: Created new class and moved style

**Review by weskubo-cgi — APPROVED** [review]

**Review by trev-dev — APPROVED** [review]

**Pull request merged**